EIPS/eip-1620.md

```diff
@@ -1,305 +1,198 @@
 ---
 eip: 1620
 title: ERC-1620 Money Streaming
-author: Paul Berg (@PaulRBerg) <hello@paulrberg.com>
+author: Paul Razvan Berg (@PaulRBerg) <paul@sablier.finance>
 discussions-to: https://github.com/ethereum/EIPs/issues/1620
-status: Draft
+status: Last Call
```

Comment:
Last call requires a `review-period-end` field. It is customary to provide at least 2 weeks for it, but given it is holiday season, I would recommend to go with at least 3-4 weeks.